### PR TITLE
Fixes for bullseye recommendations

### PR DESCRIPTION
**Pull request by dbnicholson**
Add a few packages that are recommended in bullseye that we might want. This is based on the package-recommends output and obviously is somewhat subjective.

https://phabricator.endlessm.com/T32328

### CODE REVIEW

File: os-depends [hunk 1]
```diff
@@ -130,6 +130,7 @@ xdg-user-dirs
 xdg-user-dirs-gtk
 xserver-xorg
 xserver-xorg-input-libinput
+xserver-xorg-input-wacom
```
Comment (wjt) [inline os-depends:133]: Not so sure about this one. https://phabricator.endlessm.com/T14678#663901

**Review by wjt — COMMENTED** [review]
Build failing because:

```
Package shim is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  shim-helpers-amd64-signed shim-unsigned

--- stderr ---
E: Package 'shim' has no installation candidate
eos-metapackage: error: apt-get install --no-install-recommends --download-only --dry-run --quiet fscrypt bindfs ack-grep alsa-tools ansible arcanist at awscli codecgraph evemu-tools gcovr gparted jq kbd logrotate multistrap nmap pesign pigz powertop python3-boto python3-boto3 qemu-utils resolvconf shim x11-apps returned exit code 100
-------
Package shim is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  shim-helpers-amd64-signed shim-unsigned

--- stderr ---
E: Package 'shim' has no installation candidate
eos-metapackage: error: apt-get install --no-install-recommends --download-only --dry-run --quiet fscrypt bindfs ack-grep alsa-tools ansible arcanist at awscli codecgraph evemu-tools gcovr gparted jq kbd logrotate multistrap nmap pesign pigz powertop python3-boto python3-boto3 qemu-utils resolvconf shim x11-apps returned exit code 100
-------
```

unrelated?

**Comment by dbnicholson** [comment]
I fixed the CI issue in https://github.com/endlessm/eos-meta/pull/698/commits/828f3c623f11ace818cf411fcda6f63ad1a959e4. Let me pull that into a separate PR.

File: os-depends (hunk 1 above)
Comment (dbnicholson) [reply os-depends:133]: Ah, I should have looked into this some more. My outdated understanding was that basically the only two X input drivers you want are libinput and wacom, but I think that was from before wacom support was added to libinput. I can update.

**Comment by dbnicholson** [comment]
> I fixed the CI issue in [828f3c6](https://github.com/endlessm/eos-meta/commit/828f3c623f11ace818cf411fcda6f63ad1a959e4). Let me pull that into a separate PR.

See #700.

**Comment by dbnicholson** [comment]
Removed xserver-xorg-input-wacom now.

**Comment by dbnicholson** [comment]
I added golang-github-containernetworking-plugin-dnsname per the discussion in https://github.com/endlessm/eos-build/pull/1270#discussion_r676878584.

**Review by wjt — APPROVED** [review]

**Pull request merged**